build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.6.3-6
+v1.6.3-7
```

Comment:
No, this should be cherrypicked into v1.4.
I think v1.6.3-7 is completely fine for now, it's just a new version of the image.

But we should make it a thing to document what has changed for each `vX.Y.Z-W` release, though